hyo2/ssm2/lib/formats/writers/hypack.py

```diff
@@ -33,13 +33,31 @@ def _write_header(self):
 
         header = str()
 
-        header += "FTP NEW 2\n"
-
+        # position
+        if not self.ssp.cur.meta.latitude or not self.ssp.cur.meta.longitude:
+            latitude = 0.0
+            longitude = 0.0
+        else:
+            latitude = self.ssp.cur.meta.latitude
+            longitude = self.ssp.cur.meta.longitude
+        while longitude > 180.0:
+            longitude -= 360.0
+
+        position_string = "{0:.9f} {1:.9f}".format(latitude, longitude)
+
+        # date
+        if self.ssp.cur.meta.utc_time:
+            date_string = "%s" % self.ssp.cur.meta.utc_time.strftime("%H:%M %m/%d/%Y")
+        else:
+            date_string = "%s" % datetime.datetime.now().strftime("%H:%M %m/%d/%Y")
```

Comment:
Shouldn't we use [utcnow()](https://github.com/hydroffice/hyo2_soundspeed/commit/0c09db3400a946006a6e90f28b52ba98a86caf92)?

Reply:
Yes! You are right. Good catch.

I think I might have been (blindly) inspired from the [/writers/caris.py](https://github.com/hydroffice/hyo2_soundspeed/blob/master/hyo2/ssm2/lib/formats/writers/caris.py) writer. Shouldn't both be using the [utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) method?

Looking at the description in [utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow), it looks as if the method will be deprecated as of Python 3.12. Even though I don't think it makes a difference for this use case, it would probably be worthwhile adopting now(datetime.timezone.utc) to make the datetime object timezone aware. What do you think?

```suggestion
            date_string = "%s" % datetime.datetime.now(datetime.timezone.utc).strftime("%H:%M %m/%d/%Y")
```
I could update #83 with the change above and then issue another PR updating all remaining cases of now() with now(datetime.timezone.utc). What do you think?

Reply:
It would be great!